src/LottoResult.js

```diff
@@ -0,0 +1,136 @@
+import { Console } from '@woowacourse/mission-utils';
+import { readUserInput } from './utils/readUserInput.js';
+import {
+  hasDuplicateInArray,
+  isNumber,
+  isNumberInRange,
+} from './utils/validators.js';
+import {
+  INPUT_MESSAGE,
+  LOTTO_DELIMITER,
+  LOTTO_NUMBER_END,
+  LOTTO_NUMBER_START,
+  OUTPUT_MESSAGE,
+  WINNING_NUMBER_DELIMITER,
+} from './constants.js';
+import Lotto from './Lotto.js';
+import { retryOnError } from './utils/retryOnError.js';
+
+const DEFAULT_PRIZES = [
+  { matchCount: 3, hasBonus: false, prizeAmount: 5000 },
+  { matchCount: 4, hasBonus: false, prizeAmount: 50000 },
+  { matchCount: 5, hasBonus: false, prizeAmount: 1500000 },
+  { matchCount: 5, hasBonus: true, prizeAmount: 30000000 },
+  { matchCount: 6, hasBonus: false, prizeAmount: 2000000000 },
+];
```

Comment:
`numeric separators` 를 통해 상금을 더욱 가독성 있게 표현하면 좋을 것 같아요.
```suggestion
const DEFAULT_PRIZES = [
  { matchCount: 3, hasBonus: false, prizeAmount: 5_000 },
  { matchCount: 4, hasBonus: false, prizeAmount: 50_000 },
  { matchCount: 5, hasBonus: false, prizeAmount: 1_500_000 },
  { matchCount: 5, hasBonus: true, prizeAmount: 30_000_000 },
  { matchCount: 6, hasBonus: false, prizeAmount: 2_000_000_000 },
];
```

Reply:
numeric separators 문법을 통해서 넘버 타입 숫자에 구분자를 작성할 수 있군요! 하나 배웠습니다~ 고마워요